### ISSUE TITLE
[3.0rc2] Web admin "running queries", "slow query history" pages not showing anything

### ISSUE THREAD

**Issue by ropeladder** [arangodb/arangodb#1881]
Basically what the title says. Nothing shows up when I am running a long query in the query editor and click over to the 'running queries' page. Not sure what the criteria for slow queries is but I have never seen anything there either.

Linux Mint 17.3, Firefox 46

**Comment by dothebart** [comment]
we tried to reproduce this on several browsers, using the AQL `return sleep(30)` and couldn't reproduce it.
Namely the oldest is an iceweasel 38
Does this persist in ArangoDB 3.0 for you? do you see any output in firebug?

**Comment by ropeladder** [comment]
OK I did a bit more testing, now using the full 3.0 release. I cannot reproduce using the _system database, only using existing databases (or a new empty one). The problem seems the same in Chromium.

I realized it doesn't occur right away when using a new browser instance. Here is how I can reproduce:

1. Select new test database.
2. Navigate to the queries page and run `RETURN SLEEP(30)`.
3. Click on 'Running Queries'. (the query shows up)
4. Click back on the 'Editor' tab.
5. Run `RETURN SLEEP(30)` again.
6. Click on 'Running Queries'. There are no more queries--or sometimes the query shows up briefly and then disappears (before its 30 seconds are up).

After step 6, no more queries show up in the Running Queries page unless I restart the browser (or load the _system database).
